e2e-tests/playwright/support/ui/pages/channels.ts

```diff
@@ -21,20 +21,30 @@ export default class ChannelsPage {
     readonly postDotMenu;
     readonly postReminderMenu;
     readonly deletePostModal;
+    readonly emojiGifPickerPopup;
 
     constructor(page: Page) {
         this.page = page;
-        this.postCreate = new components.ChannelsPostCreate(page.locator('#post-create'));
-        this.findChannelsModal = new components.FindChannelsModal(page.getByRole('dialog', {name: 'Find Channels'}));
+
+        // These are main sections of the page
+        // TODO: Add this.center
```

Comment:
Is there any follow up todo here? If none, then maybe good to remove?

Reply:
fixed in https://github.com/mattermost/mattermost/pull/24527